.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,15 @@
-## Pull Request Guidelines
+<!--
+Checklist for submitting a Pull Request
+
+Please remove this comment block before submitting.
 
 1. Please read our [contributor guidelines](https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md).
-1. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
-1. Follow the instructions for [labeling and writing a release note for this PR](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes) in the block below.
+2. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
+3. If you want this PR to automatically close an issue when it is merged,
+   add `fixes #<issue number>` or `fixes #<issue number>, fixes #<issue number>`
```

Comment:
@xiangpengzhao I am not changing the mechanism (lots of developers are already using this github feature), I am just documenting it. To answer your question it will close the issue that you do place in the message, so if it's wrong - the wrong issue gets closed (hopefully you notice and fix it manually).